doc/lsst.meas.base/intro.rst

```diff
@@ -0,0 +1,94 @@
+
+Typical measurement algorithms include (say) PSF fluxes, the SDSS shape algorithm, etc.
+
+The measurement driver depends on the type of measurement being performed.
+meas_base provides two: single frame measurement, which is measurement on a single visit, and forced measurement, which takes an image and an external catalog which it uses to fix source centroids and shapes.
+
+Measurement drivers are implemented as tasks, while algorithms are meas_base plugins.
+In most (but not all) cases, algorithms and drivers can be compiled fairly arbitrarily; that is, the PSF flux algorithm can be used with single frame measurement or forced measurement, etc.
+
+meas_base provides those two drivers and a bunch of simple algorithms.
+Other packages can provide more of either, following the same conventions.
+Most of our meas_extensions packages provide measurement algorithms for use in the meas_base framework (e.g. simpleShape, shapeHSM, etc).
+
+A notable extension package is ip_diffim.
+This provides a driver suitable for meas_base-style measurement on difference images and a bunch of algorithms (e.g. dipole measurement) which are of relevance in that context.
+
+Naming and selecting measurement algorithms
+===========================================
+
+The new users can generally skip over the details of measurement drivers: we have the ones we need for now, and they mostly “just work”.
+The interesting stuff all happens in algorithms.
+
+Understand first that algorithms may be referred to by strings which describe both their home package and their name.
+Thus, ``base_PsfFlux`` refers to the ``PsfFlux`` algorithm in :py:mod:`lsst.meas.base`; ``ip_diffim_DipoleFit`` refers to the ``DipoleFit`` algorithm in :py:mod:`lsst.ip.diffim`.
+
+When configuring a measurement driver (i.e. a task like :py:class:`lsst.meas.base.sfm.SingleFrameMeasurementTask`) we specify a list of plugins to be run when executing the driver.
+See, for example, `here`_.
+These can be over-ridden by the user in the task configuration.
+
+When the driver runs, it will return an ``afw::table``.
+Each row in the table contains the results of running all of the plugins which have been enabled on a particular source.
+Each algorithm plugin can add columns to the tables to contain its results.
+These columns build upon the naming scheme described above.
+Thus, if the ``base_PsfFlux`` algorithm is run, you will see a bunch of fields in the table with names starting ``base_PsfFlux_`` — ``base_PsfFlux_instFlux``, ``base_PsfFlux_instFluxErr``, and so on.
+
+(Incidentally, note that this means that the schema of the output table depends on the particular plugins that were enabled at task configuration time. This makes it awkward to provide a single document that describes pipeline outputs — they vary with user configuration.)
```

Comment:
```
.. note::

   It's impossible to statically document the schema of output tables because that schema depends on the plugins enabled by the task's configuration.
```

Then within the note, I'd add a suggestion of how to explore the schema of a table.

Reply:
I've made this into a note as suggested, but describing how to explore table schemas is outside the scope of this work.